src/app/controller/services/mapping/timestamp-mapping.service.ts

```diff
@@ -37,6 +37,7 @@ export class TimestampMappingService {
   }
 
   overlappingPublisherRoles(timestampDefinition: TimestampDefinitionTO): PublisherRole[] {
+    if(timestampDefinition){
     const userRoles = this.globals.config.publisherRoles
     const rolesForTimestamp = timestampDefinition.publisherPattern.map(p => p.publisherRole)
     return userRoles.filter((val1) => {
```

Comment:
Improper indentation!  The content should not be on the same line as the `if` guarding it.